### DISCUSSION TITLE
Pull Requests Amigáveis e inteligentes com CI-CD, Actions e Qualidade de Software

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#105] [Talks in Portuguese]
**Palestrante**: Giovana Nascimento

**Assunto**: 
Acredito que todos ficam com um frio na barriga quando o assunto e Pull request, sempre temos aquele frio na barriga quando mexemos nele, mas agora vou te apresentar boas práticas sobre isso, tornando seus pull requests mais inteligentes e melhor, de forma com ci-cd e tendo aumento de qualidade de software, implementando testes de software dentro de seus jobs

**Comment by GiovanaNp1** [comment]
[Pull Requests Amigáveis e inteligentes com CI-CD, Actions e Qualidade de Software (1).pdf](https://github.com/githubpresente/octogatosconf22/files/9575732/Pull.Requests.Amigaveis.e.inteligentes.com.CI-CD.Actions.e.Qualidade.de.Software.1.pdf)

Link da Talk com Gifs e entre outros - https://www.canva.com/design/DAFI7PXYhU4/qt0veed5uPMVzgvQG-rqZw/view?utm_content=DAFI7PXYhU4&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton
